lldb/source/Interpreter/CommandInterpreter.cpp

```diff
@@ -1302,6 +1298,36 @@ CommandObject *CommandInterpreter::GetUserCommandObject(
   return {};
 }
 
+CommandObject *CommandInterpreter::GetAliasCommandObject(
+    llvm::StringRef cmd, StringList *matches, StringList *descriptions) const {
+  std::string cmd_str(cmd);
+  auto find_exact = [&](const CommandObject::CommandMap &map) {
+    auto found_elem = map.find(std::string(cmd));
```

Comment:
```suggestion
    auto find_exact = [&](const CommandObject::CommandMap &map) {
    auto found_elem = map.find(cmd.str());
```
(in C++20 we could just pass the StringRef to `find`, but not sure that's the default version yet)

Reply:
Not yet unfortunately.